### PR TITLE
Issue 38

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: ide-eclipse.html [hunk 1]
```diff
@@ -112,12 +112,12 @@ <h5>3. Add VM arguments</h5>
...
     <div class="tab-pane active" id="nix-eclipse-non-ide-1">
 <pre class="no-border-radius"><code>
---module-path $PATH_TO_FX --add-modules=javafx.controls,javafx.fxml
+--module-path /path/to/javafx-sdk-<span class="JFX_VERSION">11</span>/lib --add-modules=javafx.controls,javafx.fxml
 </code></pre>
     </div>
     <div class="tab-pane" id="win-eclipse-non-ide-1">
 <pre><code>
---module-path %PATH_TO_FX% --add-modules=javafx.controls,javafx.fxml
+--module-path "\path\to/javafx-sdk-<span class="JFX_VERSION">11</span>\lib" --add-modules=javafx.controls,javafx.fxml
```
Comment (tiainen) [inline ide-eclipse.html]: Change / to \

File: ide-eclipse.html [hunk 2]
```diff
@@ -139,6 +139,22 @@ <h5>3. Add VM arguments</h5>
...
+<p>
+    Alternatively, you can define a global variable that can be used in future projects. Go to
+    <kbd>Preferences (File -> Settings) -> Run/Debug -> String Substitution</kbd>, and define the name 
+    of the variable as <kbd>PATH_TO_FX</kbd>, and browse to the lib folder of the JavaFX SDK to set its value, 
+    and click apply.
+    
+    <a href="images/ide/eclipse/ide/eclipse10.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse10.png" alt="String Substitution"/></a>
+</p>
+<p>
+    Then you can refer this global variable when setting the VM options as:
```
Comment (tiainen) [inline ide-eclipse.html]: ...refer *to* this...

File: ide-intellij.html [hunk 3]
```diff
@@ -111,6 +111,21 @@ <h5>4. Add VM options</h5>
...
 
+<p>
+    Alternatively, you can define a global variable that can be used in future projects. Go to
+    <kbd>Preferences (File -> Settings) -> Appearance & Behavior -> Path Variables</kbd>, and define the name 
+    of the variable as <kbd>PATH_TO_FX</kbd>, and browse to the lib folder of the JavaFX SDK to set its value, 
+    and click apply.
+    <a href="images/ide/intellij/ide/idea07.png" target="_blank"><img src="images/ide/intellij/ide/idea07.png" alt="Path Variable"/></a>
+</p>
+<p>
+    Then you can refer this global variable when setting the VM options as:
```
Comment (tiainen) [inline ide-intellij.html]: ...refer *to* this...

File: ide-intellij.html [hunk 4]
```diff
@@ -91,12 +91,12 @@ <h5>4. Add VM options</h5>
...
     <div class="tab-pane active" id="nix-idea-non-ide-1">
 <pre class="no-border-radius"><code>
---module-path $PATH_TO_FX --add-modules=javafx.controls,javafx.fxml
+--module-path /path/to/javafx-sdk-<span class="JFX_VERSION">11</span>/lib --add-modules=javafx.controls,javafx.fxml
 </code></pre>
     </div>
     <div class="tab-pane" id="win-idea-non-ide-1">
 <pre><code>
---module-path %PATH_TO_FX% --add-modules=javafx.controls,javafx.fxml
+--module-path "\path\to/javafx-sdk-<span class="JFX_VERSION">11</span>\lib" --add-modules=javafx.controls,javafx.fxml
```
Comment (tiainen) [inline ide-intellij.html]: Change / to \

File: ide-eclipse.html (hunk 1 above)
Comment (jperedadnr) [reply ide-eclipse.html]: done

File: ide-eclipse.html (hunk 2 above)
Comment (jperedadnr) [reply ide-eclipse.html]: done

File: ide-intellij.html (hunk 3 above)
Comment (jperedadnr) [reply ide-intellij.html]: done

File: ide-intellij.html (hunk 4 above)
Comment (jperedadnr) [reply ide-intellij.html]: done

**Pull request merged**